src/util.rs

```diff
@@ -93,25 +92,36 @@ pub enum Command {
     Status,
 
     // Render and display a preview
-    Preview(RenderCommand),
+    Preview{
+        #[command(subcommand)]
+        cmd: RenderCommand,
+    },
 
     // Render to an image file
     Render{
-        #[structopt(long)]
+        #[arg(long)]
         /// Image file to save render output
         file: String,
 
-        #[structopt(subcommand)]
+        #[command(subcommand)]
         cmd: RenderCommand,
     },
 
     // Print data!
-    Print(RenderCommand),
+    Print{
+        #[arg(long)]
```

Comment:
it'd be good to have a comment here for what the flag does so that appears in the CLI help? otherwise looks good (and thanks for dealing with the structopt/clap migration!)

Reply:
Good point. Added string for both the "cut" command and "chain" option. Happy to adjust them if you think they can be worded in a better way.